### DISCUSSION TITLE
How to get spotfire-mods-dev-server-1.0.1.tgz for a local npm install?

### DISCUSSION THREAD

**Discussion by jleviaguirre** [TIBCOSoftware/spotfire-mods#111] [Q&A] [answered]
A customer can't access the spotfire-mods-dev-server-1.0.1.tgz when running npm install because they have their own repository. Where can I get a hold of spotfire-mods-dev-server-1.0.1.tgz so they can run:

`npm install spotfire-mods-dev-server-1.0.1.tgz`

**Comment by objerke** [comment, marked as answer]
The .tgz file is available on npm where it is published. The 1.0.1 release can be found here: https://registry.npmjs.org/@tibco/spotfire-mods-dev-server/-/spotfire-mods-dev-server-1.0.1.tgz

The `npm view @tibco/spotfire-mods-dev-server` command shows the url of a package.
